vhdbuilder/packer/packer_source.sh

```diff
@@ -103,3 +103,15 @@ cpAndMode() {
   src=$1; dest=$2; mode=$3
   DIR=$(dirname "$dest") && mkdir -p ${DIR} && cp $src $dest && chmod $mode $dest || exit $ERR_PACKER_COPY_FILE
 }
+
+downloadDebPkgToFile() {
```

Comment:
shouldn't be here, it should be in cse_helper?

Reply:
I want to separate VHD building code (i.e here) with cse code since downloading to file is vhd-only

Reply:
moving to cse_helper